### ISSUE TITLE
Please upgrade to AWS SDK 1.11.x

### ISSUE THREAD

**Issue by alexmnyc** [dwhjames/aws-wrap#58]
Please upgrade to AWS SDK 1.11.x

**Comment by mingchuno** [comment]
@alexmnyc AWS SDK is provided depns. You can easily provide them with newer version in your own `build.sbt`

**Comment by ajr2493** [comment]
@mingchuno yes you could just upgrade the version in build.sbt, but that doesn't solve the problem of deprecation of methods as part of the latest aws release, which will require changes to this project.

**Comment by mingchuno** [comment]
@alexmnyc @ajr2493 I see, can you point me to the major difference between version 1.10.x and 1.11.x ?

**Comment by mingchuno** [comment]
@alexmnyc @ajr2493 besides, please move the discussion to https://github.com/mingchuno/aws-wrap since this repo has been dead.
